packages/types/surveys.ts

```diff
@@ -881,6 +881,9 @@ const ZSortOption = z.object({
 export type TSortOption = z.infer<typeof ZSortOption>;
 export type TSurveySummary = z.infer<typeof ZSurveySummary>;
 
+export const ZHiddenFieldValue = z.record(z.union([z.string(), z.number(), z.array(z.string())]));
```

Comment:
Same with the hierarchical structure. Also since this is sent in the js package with the response would it be better to have this in the response types?